### PR TITLE
feat: switch to react-markdown, lazy loading [TOL-501]

### PR DESCRIPTION
**Pull request by aodhagan-cf**
- Replace markdown-to-jsx with react-markdown 
- Lazily load the package when needed (lazy loading the preview tab)

### CODE REVIEW

**Review by z0al — APPROVED** [review]
Looks great 🙇🏻‍♂️

File: packages/markdown/src/index.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import './codemirrorImports';
 
 export { MarkdownEditorConnected as MarkdownEditor } from './MarkdownEditor';
-export { MarkdownPreview } from './components/MarkdownPreview';
```
Comment (z0al) [inline packages/markdown/src/index.tsx:4]: Why?

Comment (aodhagan-cf) [reply packages/markdown/src/index.tsx:4]: This was an attempt to remove it from the main bundle when lazy loading however it didn't work, want me to revert?

Comment (z0al) [reply packages/markdown/src/index.tsx:4]: It's a breaking change technically so let's revert it and revisit in the follow up

Comment (aodhagan-cf) [reply packages/markdown/src/index.tsx:4]: done!

**Pull request merged**